ui/app/components/mount-backend-form.js

```diff
@@ -88,10 +87,22 @@ export default Component.extend({
     }
   },
 
+  isModelValid(model) {
```

Comment:
nit that doesn't need to be changed, but the naming `isSomething` doesn't make me think it's a function, it seems like an attribute. Great that it returns a boolean though! Personally i would name it something like `checkModelValidity`

Reply:
this is helpful! i always get caught up on naming, and want it to be clear that it was a boolean situation (so the conditional below made sense) will change as I take naming quite seriously (...too seriously)